projects/storefrontlib/src/cms-components/product/product-tabs/product-attributes/product-attributes.component.ts

```diff
@@ -9,11 +9,30 @@ import { CurrentProductService } from '../../current-product.service';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class ProductAttributesComponent implements OnInit {
+  protected readonly PRODUCT_SCOPE =
+    this.features && this.features.isLevel('1.4')
+      ? ProductScope.ATTRIBUTES
+      : '';
+
   product$: Observable<Product> = this.currentProductService.getProduct(
-    ProductScope.ATTRIBUTES
+    this.PRODUCT_SCOPE
+  );
+
+  constructor(
+    currentProductService: CurrentProductService,
+    // tslint:disable-next-line: unified-signatures
+    features: FeatureConfigService
   );
 
-  constructor(protected currentProductService: CurrentProductService) {}
+  /**
+   * @deprecated since 1.4
+   */
+  constructor(currentProductService: CurrentProductService);
+
+  constructor(
+    protected currentProductService: CurrentProductService,
+    protected features?: FeatureConfigService
```

Comment:
given that we have the `FeatureConfigService` already available in the `CurrentProductService`, why not using the logic overthere?